label_studio/tasks/models.py

```diff
@@ -317,6 +317,12 @@ def decrease_project_summary_counters(self):
             summary = self.task.project.summary
             summary.remove_created_annotations_and_labels([self])
 
+    def save(self, *args, **kwargs):
```

Comment:
We have to check does it work on annotations delete action in DM, the most likely it won't work. Also there might be more bulk operations we should care about..

Reply:
Fixed annotations delete from DM. 
With bulk actions, I found only "create annotations from predictions" where we need this logic. In other places, we use bulk create to create tasks with annotations, so we don't need to set updated_at there.